oci/utils_unix.go

```diff
@@ -73,6 +81,12 @@ func getDevices(path, containerPath string) ([]specs.LinuxDevice, error) {
 				}
 				sub, err := getDevices(filepath.Join(path, f.Name()), cp)
 				if err != nil {
+					if errors.Is(err, os.ErrPermission) && usernsRunningInUserNS() {
```

Comment:
While writing this, I was actually wondering; would it make sense to remove the `usernsRunningInUserNS()` condition here, and just ignore permission errors here; for non-rootless these errors should (likely!) never happen, so it wouldn't make a difference.